src/display/svg.js

```diff
@@ -419,6 +421,7 @@ SVGGraphics = (function SVGGraphicsClosure() {
     this.embedFonts = false;
     this.embeddedFonts = Object.create(null);
     this.cssStyle = null;
+    this.group = null;
```

Comment:
Shouldn't we also do `this.parentGroup = null` here?

Also, let's rename `this.group` to `this._group` and `this.parentGroup` to `this._parentGroup` to indicate that they are private. This also avoids name clashes with the `group` method.